textalyze.js

```diff
@@ -17,26 +26,42 @@ function itemCounts(array) {
 /**
  * Given an input String, returns an Array containing each letter
  * @param {String} string - The String to convert to array
- * @param {Array} array - The Array containing the characters from the string
+ * @returns {Array} The Array containing the characters from the string
  */
 function arrayFrom(string) {
-  if (!string || typeof string !== 'string') {
+  if (validateString(string)) {
     return [];
   }
 
   return string.split('');
 }
-module.exports = { itemCounts, arrayFrom };
+
+/**
+ * Given an input String, returns the string with only lower case characters
+ * @param {String} string - The String to be sanitized
+ * @returns {String} The String sanitized
+ */
+function sanitize(string) {
+  if (validateString(string)) {
```

Comment:
I don't think this is a good idea, but everyone in the workshop is doing the same thing.  I want to find out why and address it together.  I'll explain why I think it's a bad idea as a group.